### PR TITLE
Update mms readmes

### PR DESCRIPTION
**Pull request by bmpotter**
- make the file different from the the service userinput format so it is not confused with that
- only read the MMS file when it is updated (some services will have to do a lot to use a new file)
- add http code checking to handle errors
- remove the userInput section from the service definition to avoid confusion about another way for HW_WHO to be set
- correct that when running the service via `hzn dev service start` the `hzn mms object ...` commands need to be explicitly directed to the local instance of MMS
- remove ess-read-all.sh because it is redundant with service.sh and https://github.com/open-horizon/anax/blob/master/test/docker/fs/hzn/service/usehello/start.sh
- add the CSS and ESS API documentation
- make the MMS file object type specific to the user (by including `$HZN_DEVICE_ID`) so when multiple people are running the sample at the same time they won't be manipulating the same MMS file.

### CODE REVIEW

File: edge/services/helloMMS/Makefile [hunk 1]
```diff
@@ -81,6 +81,11 @@ publish-only:
 	ARCH=arm64 $(MAKE) publish-service-overwrite
 	hzn exchange pattern publish -f horizon/pattern-all-arches.json
 
+publish-only-pattern-each-arch:
```
Comment (t-fine) [inline edge/services/helloMMS/Makefile]: We talked about adding `$(MAKE) publish-only-pattern-each-arch` to `publish-only` to invoke this new target

Comment (bmpotter) [reply edge/services/helloMMS/Makefile]: done

File: edge/services/helloMMS/README.md [hunk 2]
```diff
@@ -47,60 +47,59 @@ hzn exchange node confirm
```
Comment (t-fine) [inline edge/services/helloMMS/README.md]: I know this isn't part of what you edited but if you wanted to update step 3 of the Preconditions to be similar to `cpu2evtstreams` and `helloworld` and add the new step 6 to Preconditions as well:

3. As part of the infrasctucture installation process for IBM Edge Computing Manager a file called `agent-install.cfg` was created that contains the values for `HZN_ORG_ID` and the exchange and css url values. Locate this file and set those environment variables in your shell now:
```bash
eval export $(cat agent-install.cfg)
```
...
6. If you have not done so already, unregister your node before moving on:
```bash
hzn unregister -f
```

Comment (bmpotter) [reply edge/services/helloMMS/README.md]: I was wondering about this. Thx for this info! I added it.

**Pull request merged**